tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -273,7 +275,7 @@ public function testGeneratesForeignKeySqlOnlyWhenSupportingForeignKeys()
         }
     }
 
-    protected function getBitAndComparisonExpressionSql($value1, $value2)
```

Comment:
I miss the PHP 7.1 strictness.

Reply:
I don't understand, I added typehints.

Reply:
Sorry. The change is correct. I was saying that this issue wouldn't be possible on PHP 7.1 which doesn't allow loosening argument types in subclasses.